safe_cli/utils.py

```diff
@@ -41,3 +42,16 @@ def yes_or_no_question(question: str, default_no: bool = True) -> bool:
         return False
     else:
         return False if default_no else True
+
+
+def choose_option_question(
+    question: str, number_options: int, default_option: int = 0
+) -> bool:
+    if "PYTEST_CURRENT_TEST" in os.environ:
+        return 0  # Ignore confirmations when running tests
+    choices = f" [0-{number_options}] default {default_option}:"
+    reply = str(input(question + choices)).lower().strip() or str(default_option)
```

Comment:
Use `get_input` so you can easily mock the function for testing. Please add a test for this function